docs/routing/route_configuration_options.md

```diff
@@ -0,0 +1,75 @@
+# Route configuration options
+
+ ## Route domains
+ ![ILC registry domains field](../assets/domain-field.png)
```

Comment:
before screenshot add some introduction information
for example - why developers should specify some domain name. in which cases it's necessary and when it's useless. just imagine, that you are explaining someone - what is it and which value has "router domain"

Reply:
Unlike previous documentation, this has independent parts. So it should be not hard to create own description (introductional information) for each of them. So do it please for every part of the current doc.